README.md

```diff
@@ -50,6 +50,7 @@
 - 🇸🇪 Swedish
 - 🇮🇳 Tamil
 - ᴛʜ Thai
+- ᴠ🇮 Vietnamese
```

Comment:
@NattapolChan  Could you please fix the flag here?

Reply:
I can see you have updated the pull request but the flag seems to have an issue still. Could you check it?